pkg/maps/policymap/callmap.go

```diff
@@ -44,23 +45,36 @@ func (v *PlumbingValue) String() string {
 // RemoveGlobalMapping removes the mapping from the specified endpoint ID to
 // the BPF policy program for that endpoint.
 func RemoveGlobalMapping(id uint32) error {
-	gpm, err := OpenCallMap()
+	gpm, err := OpenCallMap(PolicyCallMapName)
 	if err == nil {
 		k := PlumbingKey{
 			key: id,
 		}
 		err = gpm.Map.Delete(&k)
 		gpm.Close()
 	}
+	if option.Config.EnableEnvoyConfig {
+		gpm, err2 := OpenCallMap(PolicyEgressCallMapName)
+		if err2 == nil {
+			k := PlumbingKey{
+				key: id,
+			}
+			err2 = gpm.Map.Delete(&k)
+			gpm.Close()
+		}
+		if err == nil {
+			return err2
+		}
+	}
```

Comment:
Sorry, can we move this out into the caller instead? It looks ugly having to import `github.com/cilium/cilium/pkg/option` just for this use case.

Reply:
This done as well, forgot to mention, sorry.